.github/workflows/deploy.yml

```
@@ -0,0 +1,39 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
```

I think one way to test if deployment works before merging the PR is to temporarily allow running this here. (but I am not sure if that could lead to permissions issues since the branch lives on my fork)

```suggestion
on:
  push:
    branches:
      - main
  pull_request:
    branches:
      - main
```